stubs/auth0-python/auth0/rest_async.pyi

```diff
@@ -23,7 +23,7 @@ class AsyncRestClient(RestClient):
     async def delete(self, url: str, params: dict[str, Incomplete] | None = None, data: RequestData | None = None): ...
 
 class RequestsResponse:
-    status_code: Incomplete
-    headers: Incomplete
-    text: Incomplete
+    status_code: int
+    headers: MutableMapping[str, str]
+    text: str
     def __init__(self, response, text: str) -> None: ...
```

Comment:
The implementation types the `response` param as `aiohttp.ClientResponse`. And `aiohttp.ClientResponse.headers` is typed as `CIMultiDictProxy[str]`. I'm curious what's the reason for using `MutableMapping[str, str]`

```suggestion
    headers: CIMultiDictProxy[str]
    text: str
    def __init__(self, response: aiohttp.ClientResponse, text: str) -> None: ...
```

Reply:
sorry i missed that i can import `CIMultiDictProxy` and thank you opening PR for `auth0/auth0-python`

Reply:
Ah yeah, it would require declaring a dependency in METADATA.toml.
Using `Incomplete` for now is fine.